eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/AnswerEditor/index.js

```diff
@@ -115,15 +114,6 @@ class AnswerEditor extends React.Component {
   }
 
   getAnswerTypeText(answer) {
-    if (answer.type === UNIT) {
```

Comment:
Spoke to Joe and confirmed that the answer title should change to the unit type once a selection is made.
need to put this back in. I've added a line as well to return a default text.

    if (answer.type === UNIT) {	
      if (!answer.properties.unit) {
        return answer.type;
      }
      const unitConfig = unitConversion[answer.properties.unit];	
      return (	
        <>	
          {unitConfig.unit}	
          {` (${unitConfig.abbreviation})`}	
        </>	
      );	
    }